packages/app-project/src/helpers/fetchTranslations/fetchTranslations.spec.js

```diff
@@ -5,28 +5,23 @@ import fetchTranslations from './'
 describe('helpers > fetchTranslations', function () {
   before(function () {
     nock('https://panoptes-staging.zooniverse.org/api')
+    .persist()
```

Comment:
I’d strongly advise against using `persist()`. Instead, write a `.get()` request matcher for your new test, and add it to the mocks. 

As they stand, your mocks here will match any request, and pass, even if the request query is wrong.

Reply:
With `.persist()`, you'll also need to clean up each matcher after it's no longer needed. You haven't added any clean-up code, so these request matchers will persist across all project tests.

Without `.persist()`, each request matcher is automatically deleted after it's been used once.

Reply:
Ah I see. I was initially confused about how nock responses work, and incorrectly formatted them as "this is what's available in the database" versus explicitly hardcoding what we expect to be returned from each request. I'll revert this spec file to `.get()` matchers.

Reply:
The `nock` documentation is pretty good, and I think has a section explaining how each `.get()` generates an expectation that’s done after it has been used once. It’s been a while since I looked at it, though, and my knowledge could be out-of-date.